tools/SendBlobs/BlobSender.cs

```diff
@@ -76,18 +82,21 @@ public BlobSender(string rpcUrl, ILogger logger)
             }
             ulong nonce = HexConvert.ToUInt64(nonceString);
 
-            signers.Add(new(new Signer(chainId, privateKey, logManager), nonce));
+            signers.Add(new(new Signer(chainId, privateKey, _logManager), nonce));
         }
 
         TxDecoder txDecoder = new();
+        Random random = new();
```

Comment:
```suggestion
        Random random = new(DateTime.UtcNow.Ticks);
```

Reply:
It's already done like that in  the default constructor. Problems appear mostly in tests, when several instantiations happen at the same Tick.